Assets/Scripts/ApplicationLifecycle/ApplicationController.cs

```diff
@@ -47,29 +47,31 @@ protected override void Configure(IContainerBuilder builder)
             builder.Register<ProfileManager>(Lifetime.Singleton);
 
             //these message channels are essential and persist for the lifetime of the lobby and relay services
-            builder.Register<MessageChannel<QuitGameSessionMessage>>(Lifetime.Singleton).AsImplementedInterfaces();
-            builder.Register<MessageChannel<QuitApplicationMessage>>(Lifetime.Singleton).AsImplementedInterfaces();
-            builder.Register<MessageChannel<UnityServiceErrorMessage>>(Lifetime.Singleton).AsImplementedInterfaces();
-            builder.Register<MessageChannel<ConnectStatus>>(Lifetime.Singleton).AsImplementedInterfaces();
-            builder.Register<MessageChannel<DoorStateChangedEventMessage>>(Lifetime.Singleton).AsImplementedInterfaces();
+            builder.RegisterInstance(new MessageChannel<QuitGameSessionMessage>()).AsImplementedInterfaces();
```

Comment:
```suggestion
            // Registering as instance to prevent code stripping on iOS
            builder.RegisterInstance(new MessageChannel<QuitGameSessionMessage>()).AsImplementedInterfaces();
```

Let me know if that's right

Reply:
That is right. Didn't find another way to deal with this otherwise yet.